src/bitmessagemain.py

```diff
@@ -16,10 +16,12 @@
 os.chdir(app_dir)
 sys.path.insert(0, app_dir)
 
+
```

Comment:
Import should be placed in a order.
Please maintain orders
* import statements first
* from statements should be next.

Reply:
It's impossible in this module. PyBitmessage won't start after such changes.